src/services/feePolicy/__tests__/feePolicyService.pbt.test.ts

```
@@ -0,0 +1,357 @@
+              currencies: [
+                {
+                  currency: "NGN",
+                  targetWeight,
+                  actualWeight,
+                },
+              ],
+            });
+
+            const fee = await getBurnFeeBps("NGN");
+            expect(fee).toBe(expectedFee);
+          }
```

ESLint

[error] Insert `,`

(prettier/prettier)